welcomebot.go

```diff
@@ -134,6 +141,13 @@ func respondToJoin(rtm *slack.RTM, ev *slack.MessageEvent, name string, config C
 		}
 	}
 
+	for _, ephResponse := range config.EphResponses {
+		if ephResponse.Channel == name {
+			log.Infof("Sending ephemeral reply to %s in channel %s", ev.User, name)
+			postEphemeral(rtm, ev.Msg.Channel, ev.User, ephResponse.Response, ephResponse.Raw)
```

Comment:
```
$ go run welcomebot.go
# command-line-arguments
./welcomebot.go:147:4: undefined: postEphemeral
```

Reply:
A minor point 😂 
I took a punt based off [nlops/slack chat.go](https://github.com/nlopes/slack/blob/d8ac454a777b2593d818e3e634bb99245962c42b/chat.go#L111)

Reply:
Oh! I know what I need to do...
I was replacing `sendMessage` with `postEphemeral` (should be `PostEphemeral` anyway, confusing it with the imorted `SendMessage`. `sendMessage` is the internal function whereas `SendMessage` is imported